devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install nginx
+  yum: name=nginx state=present
+
+- name: http service state
+  service: name=nginx state=started enabled=yes
+...
+
+
+
+#### Check if nginx is installed ####
+# - name: Check nginx install
+#   changed_when: false
+#   yum: list=nginx.x86_64
+#   register: check_install_result
+
+# - name: Print IP Address
+#   debug: var=check__install_result.stdout_lines
+
+#### Allow port access - currently access denied ####
+# - name: insert firewalld rule for nginx
+#   become: true
+#   firewalld: port={{ nginx_port }}/tcp permanent=true state=enabled immediate=yes
+#   ignore_errors: yes
```

Comment:
I think I was getting an error like access denied. I have that block in the newer script I've been testing and it seems to be ok.